common/video.c

```diff
@@ -190,25 +179,34 @@ videoGetLine(lua_State *L, int index, Line *line)
 	line->y2 = tableGetInt(L, index, "y2");
 }
 
+/*
+ * Deprecated in favor of videoGetColorRGB and SDL_MapRGBA, which enforce a
+ * common representation of color values across color tables and hexadecimal
+ * formats.
+*/
 Uint32
 videoGetColorHex(lua_State *L, int index)
 {
-	Uint32 value = 0;
+	/*Uint32 value = 0;
```

Comment:
Go ahead and delete this commented-out code outright; if you ever need the old version it'll be in the Git history.